extensions/github/src/api/githubClient.ts

```diff
@@ -20,7 +20,7 @@ function onAuthorize({ token, type }: { token: string; type: string }) {
 
 export const githubOAuthService = OAuthService.github({
   personalAccessToken: preferences.personalAccessToken,
-  scope: "notifications repo read:org read:user read:project",
```

Comment:
This action requires our users to re-authorize their accounts for more permissions. You may need to prompt the user to the re-authorize page.

Reply:
This might be an issue when someone wants to close/re-open a project, I already added a test for the scopes in there. If the user tries to for example close the project, but lacks the required scopes, it will show a failureToast with the message to reauthenticate.